.ladle/adapters/PlainComponentAdapter.tsx

```diff
@@ -18,6 +20,17 @@ import type {
 import type { ComponentsContextType } from '../../src/contexts/ComponentAdapter/ComponentsProvider'
 
 export const PlainComponentAdapter: ComponentsContextType = {
+  Alert: ({ label, children, variant = 'info', icon }: AlertProps) => {
```

Comment:
Did we settle on a name for variant? I wonder if something like `status` might be more effective for conveying the `info` `warning` `success` etc that will be repeated across other components that use this hierarchy. Variant ends up being kinda general

Reply:
Fixed!